guides/_subsections/download-edit-and-upload-a-customized-event-template.md

```diff
@@ -51,79 +51,79 @@ definition to download.
       summary="Navigate to Events"
       image-name="2.3.0/navigate-to-events-1.png"
       caption="
-        Supply JMX credentials to authenticate to the target, if necessary. If
-        the target is not configured with JMX authentication then the
+        Supply <b>JMX</b> credentials to authenticate to the target, if necessary. If
+        the target is not configured with <b>JMX</b> authentication then the
         connection attempt will continue without prompting for credentials.
       "
       text=navigate-to-events-include-text
     %}
   </li>
   <li>
     {% include howto_step.html
-      summary="Download the Template"
+      summary="Download the <i>Template</i>"
       image-name="2.3.0/download-edit-and-upload-a-customized-event-template-1.png"
       caption="
-        Click the action overflow three-dot menu on the right side of the
+        Click the action overflow \"&#65049;\" <i>three-dot</i> menu on the right side of the
         template entry in the table, then click <i>Download</i>.
       "
     %}
   </li>
   <li>
     {% capture edit-template-additional-content %}
-      The template definition is an XML file, so you may open and edit it
-      with your favourite text editor or XML document editor. The JMC
-      Template Manager can also be used for this purpose, as described
+      The template definition is an <code>XML</code> file, so you may open and edit it
+      with your favourite text editor or <code>XML</code> document editor. The <b>JMC
+      Template Manager</b> can also be used for this purpose, as described
       <a href="{{ page.url }}#edit-template-with-jmc">
         in this section
```

Comment:
Can we replace this with "as described in [Edit Template With JMC](http://127.0.0.1:4000/guides/#edit-template-with-jmc)"